### ISSUE TITLE
Try different loss functions

### ISSUE THREAD

**Issue by AlessandroVol23** [AlessandroVol23/predict_ico_success_lmu#100]
https://catboost.ai/docs/concepts/loss-functions-classification.html

- Cross Entropy
- Maximize mcc directly
